catt/cli.py

```diff
@@ -190,60 +190,54 @@ def process_subtitle(ctx, param, value):
 @click.pass_obj
 def cast(settings, video_url, subtitle, force_default, random_play, no_subs):
     controller = "default" if force_default else None
+    subtitle_url = None
+    playlist_playback = False
     cst, stream = setup_cast(settings["device"], video_url=video_url,
                              prep="app", controller=controller)
 
     if stream.is_local_file:
         click.echo("Casting local file %s..." % video_url)
-        click.echo("Playing %s on \"%s\"..." % (stream.video_title, cst.cc_name))
-        if subtitle is None and no_subs:
-            subtitle_url = None
-        else:
+        if not (subtitle is None and no_subs):
```

Comment:
Would this be clearer as `if subtitle or not no_subs`?

Reply:
It would.